.changeset/unlucky-windows-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
+---
+
+add Harmony testnet
```

Comment:
I think that you can remove one of the changeset files as it is duplicated

Reply:
I deleted one, I hope it will be enough ; thanks